.gitignore

```diff
@@ -15,6 +15,7 @@ user_playbooks
 playbooks/galaxy_roles
 requirements.yml
 inventories/local_*
+roles/customize_home/files/
```

Comment:
What do you think about instead supporting these two options:

 1) a git repository that can be synced
 2) user specifying a variable file that includes a list of all files on their control node to copy over, that file could live in a `user_variables` folder at the top level to match the `user_playbooks` directory.

Reply:
> a git repository that can be synced

I could see a use case for the git repository (that is where I keep my custom dotfiles and setup scripts), let me look into that.

> user specifying a variable file that includes a list of all files on their control node to copy over, that file could live in a user_variables folder at the top level to match the user_playbooks directory.

My thought was, to keep things simple, you can just symlink anything on your machine to `/roles/customize_home/files`. I worry about adding too much complexity here before we have gotten any feedback, but I'll think it over.

Reply:
Can this then be moved elsewhere? I don't like ignoring a directory that deep in our roles as I've had bad experiences with it as a developer.

Reply:
Sorry, by elsewhere I mean a directory we ignore at the top level like `user_dotfiles` or `user_devel_environment_files`